docs/dev/connectors/streamfile_sink.zh.md

```diff
@@ -705,6 +705,9 @@ Hadoop 2.7 之前的版本不支持这个方法，因此 Flink 会报异常。
 <span class="label label-danger">重要提示 3</span>: Flink 以及 `StreamingFileSink` 不会覆盖已经提交的数据。因此如果尝试从一个包含 in-progress 文件的旧 checkpoint/savepoint 恢复，
 且这些 in-progress 文件会被接下来的成功 checkpoint 提交，Flink 会因为无法找到 in-progress 文件而抛异常，从而恢复失败。
 
+<span class="label label-danger">重要提示 4</span>: Flink 以及 `StreamingFileSink` 不会覆盖已经提交的数据。因此如果尝试从一个包含 in-progress 文件的旧 checkpoint/savepoint 恢复，
```

Comment:
Sorry for correcting you on the Chinese documentation, but this seems to be wrong as it looks like a copy of the point 3 above. I hope I am correct :P

Reply:
You are right. :P 
I think it might be the md preview plugin that leads to the problem.
I preview the result of the document using the md plugin. But after previewing it suspends the whole IntelliJ. I just kill the IntelliJ and commit the result and push the branch. :(( Maybe some changes are lost because of the killing.